webpack.config.js

```diff
@@ -0,0 +1,78 @@
+const path = require("path");
+
+const {CleanWebpackPlugin} = require("clean-webpack-plugin");
+const HtmlWebPackPlugin = require("html-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+
+module.exports = {
+  mode: "development",
+
+  entry: {
+    main: "./front-end/src/index.tsx"
+  },
+
+  output: {
+    filename: "[name].js",
+    path: path.resolve(__dirname, "build")
+  },
+
+  devServer: {
+    open: true,
+    port: 9000
```

Comment:
```suggestion
    port: process.env.WEBPACK_PORT || 9000
```

It's a good practice to allow overriding the default port if it's colliding if another service.
You do it like that then:
```
$ WEBPACK_PORT=7000 npm start
```